.travis.yml

```diff
@@ -13,7 +13,8 @@ script:
     - make package
     - make package-clean
 after_success:
-    coveralls
+    - "curl -X POST -d \"token=${readthedocs_token}\" \"https://readthedocs.org/api/v2/webhook/nailgun/76623/\""
```

Comment:
What is this number (76623)? Where does it come from? readthedocs documentation says it's `<id>`, but nothing more. I'm wondering if this is dynamic or static.

Reply:
It's static project no. also I might change the implementation to use authenticated webhook